java/main/src/main/java/com/saucelabs/saucebindings/CITools.java

```diff
@@ -56,8 +52,11 @@ private static void storeBuildInfo() {
    * @return the constant name of the build
    */
   public static String getBuildName() {
-    storeBuildInfo();
-    return buildName;
+    if (!Objects.equals(getCiToolName(), "unknown")) {
```

Comment:
Why not `equalsIgnoreCase`? This compares strings, while `equals` compares hash codes.

Reply:
Good call

Reply:
https://github.com/saucelabs/sauce_bindings/pull/334/commits/739a24eb03b71261fcd7608f2fb3ec70ce7541af